src/logger.cpp

```diff
@@ -0,0 +1,87 @@
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <locale>
+#include <sstream>
+
+#include "logger.h"
+
+namespace ohl::logger {
+
+const char* logPath = "ohl.log";
+
+/**
+ * @brief Logs a logMessage to the log file ohl::logger::logPath. \n will be
+ * added at the end
+ *
+ * @param logMessage the message to be logged.
+ */
+void log( const char * logMessage ) {
```

Comment:
My philosophy with documentation:
- If the function's in the header, document it there
- If it's not, document at the function definition (rather than at a forward reference)

A function should never be documented twice.

A very minor nitpick: I'd've gone with `@note Automatically adds newlines`, keep the brief briefer.